Documentation/PackedAssetBuilds.md

```diff
@@ -1,64 +1,126 @@
 
-### Build profile build and process assets
+### Build o3de-multiplayersample and the engine (non-monolithic)
+
+Build a regular profile build of the game as per the [README.md](../README.md) in an engine-centric way. Be sure to build from the source engine and not the installer. Here is an example:
+
+Clone the engine from source.
+
+```shell
+C:\git> git clone https://github.com/o3de/o3de
+```
+
+Navigate to `C:\git\o3de`.
+Create a build folder for non-monolithic build of the engine, for example `C:\git\o3de\build_non_mono`.
+
+```shell
+C:\git\o3de> mkdir build_non_mono
+```
+> The location and the name of the build folder is optional.
 
-Build and run MPS as per the [README.md](../README.md) and ensure all assets are built.
+Configure the engine in a non-monolithic mode with o3de-multiplayersample project.
 
-### Test the profile build
+```shell
+C:\git\o3de> cmake -S . -B build_non_mono -DLY_MONOLITHIC_GAME=OFF -DLY_PROJECTS="C:\git\o3de-multiplayersample"
+```
+
+> `-DLY_MONOLITHIC_GAME=OFF` is the default value but for clarity it's specified here explicitly.
+
+Build the Editor. This will compile the project and necessary gems to produce the required game assets.
+
+```shell
+C:\git\o3de> cmake --build build_non_mono --target Editor --config profile
+```
+
+Run the Asset Processor with o3de-multiplayersample and let all the assets get processed.
 
-* Open the game in editor
-    * load `NewStarBase` level
-    * Verify that game can launch and connect to local server from editor
-* Validate local game launcher can connect to local server
+```shell
+C:\git\o3de> .\build_non_mono\bin\profile\AssetProcessor.exe --project-path C:\git\o3de-multiplayersample
+```
 
 ### Build AssetBuilder
 
-You will need to build the [AssetBundler](https://www.o3de.org/docs/user-guide/packaging/asset-bundler/overview/) tool if not built.
+You will need to build the [AssetBundler](https://www.o3de.org/docs/user-guide/packaging/asset-bundler/overview/) tool if not built to create game bundles (.pak files).
 
 For example:
 ```shell
-cmake --build build\windows --target AssetBundler --config profile -- /m /nologo
+C:\git\o3de> cmake --build build_non_mono --target AssetBundler --config profile
 ```
 
 ### Build monolithic game
 
 Build a second version of the executables as monolithic pak builds.
 
-``` shell
+> A separate build folder is required for building monolithic binaries, separate from the non-monolithic build folder.
+
+```shell
+C:\git\o3de> mkdir build_mono
+
 # Create build files for a monolithic build that also disables all user/project registry settings overrides
-cmake -B build\windows_mono -S . -G "Visual Studio 16" -DLY_3RDPARTY_PATH=c:\your\path\to\3rdParty -DLY_MONOLITHIC_GAME=1 -DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0
+C:\git\o3de> cmake -S .. -B build_mono -DLY_MONOLITHIC_GAME=1 -DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0 -DLY_PROJECTS="C:\git\o3de-multiplayersample"
```

Comment:
Just wanted to confirm you meant to write ".." here?

Reply:
You are correct. It should be a single dot.